app/core/src/main/java/com/fsck/k9/message/PgpMessageBuilder.java

```diff
@@ -109,15 +111,27 @@ protected void buildMessageInternal() {
             return;
         }
 
+        addAutocryptHeaderIfAvailable(openPgpKeyId);
+        addDraftStateHeader();
```

Comment:
Will this add the `Autocrypt-Draft-State` header to all messages, regardless of whether or not it's a draft message?

Reply:
it did. weird, I had it right in an earlier commit. :)